utils.go

```diff
@@ -0,0 +1,18 @@
+package main
```

Comment:
I believe the original problem here was a misunderstanding on Brian's part of this being an importable package as opposed to a binary. `package main` was always the correct choice. 

@BenjaminTrapani , you should always feel free to make PR comments a discussion before acting on them...it's the best way to both save time and produce the best code in the end. Either asking "Does it make sense to create a whole new package for one function?" or just pointing out that this was a `main` package to begin with would have been  effective questions.